cognite/client/data_classes/postgres_gateway/users.py

```diff
@@ -0,0 +1,148 @@
+from __future__ import annotations
+
+from abc import ABC
+from dataclasses import dataclass
+from typing import TYPE_CHECKING, Any, Literal, NoReturn
+
+from typing_extensions import Self
+
+from cognite.client.data_classes._base import (
+    CogniteObject,
+    CognitePrimitiveUpdate,
+    CogniteResource,
+    CogniteResourceList,
+    CogniteUpdate,
+    PropertySpec,
+    WriteableCogniteResource,
+    WriteableCogniteResourceList,
+)
+
+if TYPE_CHECKING:
+    from cognite.client import CogniteClient
+
+
+@dataclass
+class SessionCredentials(CogniteObject):
+    nonce: str
+
+    @classmethod
+    def _load(cls, resource: dict[str, Any], cognite_client: CogniteClient | None = None) -> Self:
+        return cls(
+            nonce=resource["nonce"],
+        )
```

Comment:
I feel like this whole object is unnecessary and that `UserWrite` could hold the nonce directly.

Reply:
I do agree, but happens if the spec is extended?

Reply:
➕

Reply:
Just to be clear, the spec does not have a concept of User, just expects a list of `SessionCredentials`. That said, I'm not against us adding it.